packages/node_modules/@webex/plugin-meetings/src/members/util.js

```diff
@@ -136,6 +136,11 @@ MembersUtil.generateRaiseHandMemberOptions = (memberId, status, locusUrl) => ({
   locusUrl
 });
 
+MembersUtil.generateLowerAllHandsMemberOptions = (requestingParticipantId, locusUrl) => ({
```

Comment:
issue: This isn't currently covered by any test - the index test just checks this (and consecutive functionality) is called, and the request test only checks the `getLowerAllHandsMemberRequestParams` output.